lib/shopify_app/controller_concerns/token_exchange.rb

```diff
@@ -5,8 +5,18 @@ module TokenExchange
     extend ActiveSupport::Concern
     include ShopifyApp::AdminAPI::WithTokenRefetch
 
+    INVALID_SHOPIFY_ID_TOKEN_ERRORS = [
+      ShopifyAPI::Errors::CookieNotFoundError,
```

Comment:
Would we ever have a cookie present for token exchange? If it's only meant to run for embedded apps there should never be any cookies involved, right?

Reply:
Yea cookies are never involved.. but the way `ShopifyAPI::current_session_id` is setup, it'll throw a "CookieNotFoundError" if we pass in a nil auth header (which will be the case for the initial server load when we're not using id_token from URL params), so once I change this to use the new util method, we won't be catching this `CookieNotFoundError` but `MissingJwtTokenError` instead.

Reply:
Yea exactly :(